meltano/edk/extension.py

```diff
@@ -23,8 +23,13 @@ class DescribeFormat(str, Enum):
 class ExtensionBase(metaclass=ABCMeta):
     """Basic extension interface that must be implemented by all extensions."""
 
-    def pre_invoke(self) -> None:
-        """Called before the extension is invoked."""
+    def pre_invoke(self, invoke_name: str | None, *invoke_args: Any) -> None:
```

Comment:
@pandemicsyn what is responsible for calling `pre_invoke` and `post_invoke`?

Reply:
Gets called two ways.

1. [edk.pass_through_invoker](https://github.com/meltano/edk/blob/main/meltano/edk/extension.py#L91 ) which is used by the cookiecutter generated `pass_through_cli` that get's configured for wrappers through which all wrapped calls pass: (https://github.com/meltano/airflow-ext/blob/main/airflow_ext/pass_through.py). 
2. The other spot is from explicit user controlled calls from within custom commands. e.g. the superset extension may call pre_invoke only when "add_user" , or "initialize" are called to ensure that the system db exists.

Reply:
@pandemicsyn Doesn't `ExtensionBase.pass_through_invoker` need to be updated by this PR? `invoke_name` is not an optional argument.

Reply:
Doh, yea - you're right.